tko_web_sessions_management/main.py

```diff
@@ -75,6 +75,9 @@ def web_login(self, redirect=None, **kw):
             if 'login' in request.params and 'password' in request.params:
                 uid = request.session.authenticate(request.session.db, request.params[
                                                    'login'], request.params['password'])
+            reason1 = False
+            reason2 = False
+            reason3 = False
```

Comment:
Yes, you have some reason. But don't tell the user what happen with generate support tickets.

I have remove it.